scopes/explorer/ui/base-component-card/card/card.module.scss

```diff
@@ -0,0 +1,15 @@
+.componentCard {
```

Comment:
consider using base-ui.surfaces.card

Reply:
card has built in padding that I will need to override. lets talk about this later